Jenkinsfile.promote

```diff
@@ -107,17 +108,30 @@ pipeline {
                 }
             }
         }
-        stage('Git tag') {
+        stage('Git tag and release CLI') {
             when {
                 expression {
-                    return getGitTag() != ""
+                    return isRelease() && getGitTag() != ""
                 }
             }
             steps {
                 script {
-                    sh "git pull origin ${getBuildBranch()}"
-                    githubscm.tagRepository(getGitTag(), env.BUILD_TAG)
-                    githubscm.pushObject("origin", "--tags ${getGitTag()}")
```

Comment:
Just noticed, those should be deleted or ?

Reply:
Yes, because `github-release release` creates the tag. On my bot repo, there was no `7.15.0` tag before I ran the build.

Reply:
oh ok :)